app/components/teams/team/member.hbs

```diff
@@ -2,12 +2,12 @@
   class="text-center"
   data-test-member={{@member.name}}
 >
-  <ResponsiveImage
-    @src="/images/team/{{@member.image}}"
-    @width={{100}}
+  <img
```

Comment:
@simonihmig

Hello. I noticed that, after upgrading `ember-source` to `3.25.4`, the `<ResponsiveImage>` component results in the following error when tests are run locally (I didn't see an issue in CI):

```
undefined is not a function@ 118 ms
Source:   
TypeError: undefined is not a function
    at compilable (http://localhost:7357/assets/vendor.js:44189:42)
    at TemplateImpl.asLayout (http://localhost:7357/assets/vendor.js:44894:28)
    at ConstantsImpl.resolvedComponent (http://localhost:7357/assets/vendor.js:45172:53)
    at resolveComponent (http://localhost:7357/assets/vendor.js:41950:22)
    at encodeOp (http://localhost:7357/assets/vendor.js:44296:18)
    at pushOp (http://localhost:7357/assets/vendor.js:44220:7)
    at http://localhost:7357/assets/vendor.js:43787:7
    at Compilers.compile (http://localhost:7357/assets/vendor.js:42232:7)
    at compileStatements (http://localhost:7357/assets/vendor.js:44224:17)
    at maybeCompile (http://localhost:7357/assets/vendor.js:44203:18)
```

Could you look into why this may be? In the meantime, would it be all right with you if I revert the invocation to use `<img>` tag?